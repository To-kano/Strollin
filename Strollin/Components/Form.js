import React from 'react';

import { Input } from 'react-native-elements';

import {
  Text, View
} from 'react-native';

function Form({
  style, title, keyboardType, value, onChangeText, placeholder
}) {
  return (
    <View style={style}>
      <Text style={{ color: 'grey' }}>{title}</Text>
      <Input
        autoCapitalize="none"
        keyboardType={keyboardType}
        style={{ marginTop: 200, marginHorizontal: 40, height: 40 }}
        placeholder={placeholder}
        value={value}
        onChangeText={(valueText) => {
          onChangeText(valueText);
        }}
      />
    </View>
  );
}

export default Form;
