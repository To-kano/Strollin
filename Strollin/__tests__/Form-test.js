import React from 'react';
import {shallow} from 'enzyme';
import Form from '../Components/Form';

const dataTest = {
    name: 'test',
    budget: '10',
    period: '10 am to 10 pm',
    destinations: [
        '1',
        '2',
        '3',
    ]
}

describe('Form', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const component = shallow(<Form data={dataTest}/>)
            expect(component).toMatchSnapshot()
        });
    });
});