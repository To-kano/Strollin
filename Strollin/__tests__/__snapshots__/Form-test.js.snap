// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
